<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>eCCM</class>
 <widget class="QWidget" name="eCCM">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1262</width>
    <height>778</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>eCCM</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <widget class="rawPreviewWidget" name="rawPreviewWidget" native="true">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(100, 100, 100);</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0,0,1,0,0,0,0,1,0,1">
     <item>
      <widget class="QPushButton" name="importRawBtn">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(170, 170, 255);
</string>
       </property>
       <property name="text">
        <string>import raw</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>patch size:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="patchSizeSlider">
       <property name="maximum">
        <number>80</number>
       </property>
       <property name="pageStep">
        <number>5</number>
       </property>
       <property name="value">
        <number>80</number>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="minimumSize">
        <size>
         <width>166</width>
         <height>209</height>
        </size>
       </property>
       <property name="title">
        <string>adjust ROI</string>
       </property>
       <widget class="QPushButton" name="adjTopOuterBtn">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>20</y>
          <width>75</width>
          <height>24</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(128, 128, 128);</string>
        </property>
        <property name="text">
         <string>^</string>
        </property>
       </widget>
       <widget class="QPushButton" name="adjLeftOuterBtn">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>80</y>
          <width>21</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(128, 128, 128);</string>
        </property>
        <property name="text">
         <string>&lt;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="adjRightOuterBtn">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>80</y>
          <width>21</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(128, 128, 128);</string>
        </property>
        <property name="text">
         <string>&gt;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="adjBottomOuterBtn">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>180</y>
          <width>75</width>
          <height>24</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(128, 128, 128);</string>
        </property>
        <property name="text">
         <string>v</string>
        </property>
       </widget>
       <widget class="QPushButton" name="adjTopInnerBtn">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>50</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(128, 128, 128);</string>
        </property>
        <property name="text">
         <string>v</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>150</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(128, 128, 128);</string>
        </property>
        <property name="text">
         <string>^</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>80</y>
          <width>21</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(128, 128, 128);</string>
        </property>
        <property name="text">
         <string>&gt;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>111</x>
          <y>80</y>
          <width>20</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(128, 128, 128);</string>
        </property>
        <property name="text">
         <string>&lt;</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="previewBrightLabel">
       <property name="text">
        <string>preview brightness:1.0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="previewBrightSlider">
       <property name="maximum">
        <number>40</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="pageStep">
        <number>2</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="importTargetBtn">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(128, 128, 128);</string>
       </property>
       <property name="text">
        <string>import target&amp;gamma</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="calcuateCcmBtn">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(85, 170, 127);</string>
       </property>
       <property name="text">
        <string>calculate ccm</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>rawPreviewWidget</class>
   <extends>QWidget</extends>
   <header>rawpreviewwidget.h</header>
   <container>1</container>
   <slots>
    <slot>adjRoiOuterTop()</slot>
    <slot>adjRoiOuterBottom()</slot>
    <slot>adjRoiOuterLeft()</slot>
    <slot>adjRoiOuterRight()</slot>
    <slot>adjRoiInnerTop()</slot>
    <slot>adjRoiInnerLeft()</slot>
    <slot>adjRoiInnerBottom()</slot>
    <slot>adjRoiInnerRight()</slot>
    <slot>previewBrightAdj(int)</slot>
    <slot>patchSizeChange(int)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>importRawBtn</sender>
   <signal>clicked()</signal>
   <receiver>eCCM</receiver>
   <slot>importRawFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1179</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel">
     <x>1119</x>
     <y>-21</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>adjTopOuterBtn</sender>
   <signal>clicked()</signal>
   <receiver>rawPreviewWidget</receiver>
   <slot>adjRoiOuterTop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1185</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>1072</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>adjBottomOuterBtn</sender>
   <signal>clicked()</signal>
   <receiver>rawPreviewWidget</receiver>
   <slot>adjRoiOuterBottom()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1161</x>
     <y>384</y>
    </hint>
    <hint type="destinationlabel">
     <x>1083</x>
     <y>384</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>adjLeftOuterBtn</sender>
   <signal>clicked()</signal>
   <receiver>rawPreviewWidget</receiver>
   <slot>adjRoiOuterLeft()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1094</x>
     <y>313</y>
    </hint>
    <hint type="destinationlabel">
     <x>1083</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>adjRightOuterBtn</sender>
   <signal>clicked()</signal>
   <receiver>rawPreviewWidget</receiver>
   <slot>adjRoiOuterRight()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1238</x>
     <y>324</y>
    </hint>
    <hint type="destinationlabel">
     <x>1083</x>
     <y>450</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>adjTopInnerBtn</sender>
   <signal>clicked()</signal>
   <receiver>rawPreviewWidget</receiver>
   <slot>adjRoiInnerTop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1160</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>1076</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>rawPreviewWidget</receiver>
   <slot>adjRoiInnerLeft()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1134</x>
     <y>318</y>
    </hint>
    <hint type="destinationlabel">
     <x>909</x>
     <y>354</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>rawPreviewWidget</receiver>
   <slot>adjRoiInnerBottom()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1175</x>
     <y>355</y>
    </hint>
    <hint type="destinationlabel">
     <x>1071</x>
     <y>432</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>rawPreviewWidget</receiver>
   <slot>adjRoiInnerRight()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1205</x>
     <y>311</y>
    </hint>
    <hint type="destinationlabel">
     <x>1044</x>
     <y>521</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>previewBrightSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>rawPreviewWidget</receiver>
   <slot>previewBrightAdj(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1194</x>
     <y>437</y>
    </hint>
    <hint type="destinationlabel">
     <x>1067</x>
     <y>567</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>previewBrightSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>eCCM</receiver>
   <slot>showPreviewBright(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1248</x>
     <y>438</y>
    </hint>
    <hint type="destinationlabel">
     <x>1262</x>
     <y>493</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>patchSizeSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>rawPreviewWidget</receiver>
   <slot>patchSizeChange(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1220</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>1071</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>calcuateCcmBtn</sender>
   <signal>clicked()</signal>
   <receiver>eCCM</receiver>
   <slot>calculateCCM()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1174</x>
     <y>653</y>
    </hint>
    <hint type="destinationlabel">
     <x>1263</x>
     <y>715</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>importTargetBtn</sender>
   <signal>clicked()</signal>
   <receiver>eCCM</receiver>
   <slot>importTargetFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1237</x>
     <y>512</y>
    </hint>
    <hint type="destinationlabel">
     <x>1270</x>
     <y>603</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>importRawFile()</slot>
  <slot>showPreviewBright(int)</slot>
  <slot>calculateCCM()</slot>
  <slot>importTargetFile()</slot>
 </slots>
</ui>
